runtime/src/accounts_db.rs

```diff
@@ -6977,15 +6977,17 @@ impl AccountsDb {
         mut stats: HashStats,
```

Comment:
there are HashStats here already used. Maybe these belong in there? The individual pieces inside 'calc_time' are already broken out. Look at metric "calculate_accounts_hash_without_index"